website/docs/business-case/BusinessCaseIndustry.md

```diff
@@ -0,0 +1,151 @@
+* *References.* [EU Carbon Border Adjustment Mechanism](https://taxation-customs.ec.europa.eu/carbon-border-adjustment-mechanism_en#where-to-report), [Wood MacKenzie CBAM Analysis](https://www.woodmac.com/news/opinion/how-will-the-eus-cbam-impact-global-iron--steel/)
+
+### Finance Costs
+
+UNTP provides a framework based on international standards which can accommodate different ESG risks, enabling development banks to standardize their reporting and ensuring their mandate, without having to create ad-hoc structures for each Sustainable Supply Chain Finance Deal. This unlocks a significant trade finance gap, and enables preferential finance to reach deep-tier suppliers. Access to lower financing costs for suppliers results in lower cost of goods sold and improved margins. These trade finance arrangements often come with grants that can support costs associated with the ESG transition, such as support certification, consulting or implementation of new ERP systems for reporting.
+
+#### Access to Trade Finance
+The Asian Development Bank (ADB) estimates that the global trade finance gap was approximately $2.5 trillion in 2022, up from $1.5 trillions in 2016 with a significant portion attributable to SMEs applicants, lack of visibility, and issues with country risk, credit-worthiness and lack of sufficient information by the applicant. At the same time, Supply Chain Finance (SCF) has grown from $330 billion in 2015 to $1.8 trillion in 2021, despite this growth, SCF has has not yet had a major impact in reducing the trade finance gap due to difficulty reaching past tier 1 suppliers. By adopting the UNTP, this gap can be reduced by enabling more companies to access preferential financing thanks to increased visibility over ESG credentials and ability to provide identity assurance from a trusted register, combined with SCF reverse factoring operating models which reduce applicants risk by tying the financing to the buyer credit risk.
+
+* *References* Asian Development Bank (ADB), [Trade Finance Gaps Growth and Jobs Survey 2021](https://www.adb.org/publications/2021-trade-finance-gaps-growth-jobs-survey), 
+ [Trade finance gaps growth jobs survey 2023](https://www.adb.org/publications/2023-trade-finance-gaps-growth-jobs-survey). [Deep-Tier Supply Chain Finance 2022](https://www.adb.org/publications/deep-tier-supply-chain-finance)
+
+#### Reduced Finance Costs
+
+According to the International Finance Corporation (IFC), companies that adopt sustainable practices can reduce their financing costs by up to 20% due to lower risk premiums and better access to capital.
+
+* *References* International Finance Corporation (IFC), "Sustainable Finance: Creating Value for Companies and Investors," 2020.
```

Comment:
I've scanned the internet, and even checked every 2020 publication of IFC, but cannot find this publication..